clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupOptions.java

```diff
@@ -32,12 +32,23 @@
 public class RemoveMembersFromConsumerGroupOptions extends AbstractOptions<RemoveMembersFromConsumerGroupOptions> {
 
     private Set<MemberToRemove> members;
```

Comment:
Could we just make members to be `Optional<Set<MemberToRemove>>` so that we don't need a separate removeAll parameter?

Reply:
Sure. Taking a step further, can we just keep the the type `Set<MemberToRemove>` for members unchanged and treat it as `removeAll` if the `members` is empty set?

Reply:
Updated~